### ISSUE TITLE
As Idem I want to update my digital surge referral link within the app

### ISSUE THREAD

**Issue by MikeyRobertson** [dltxio/idem-mobile#294]
As Idem I want to update my digital surge referral link within the app

Full referral link - https://digitalsurge.com.au/?ref=RW21V
## AC
- Given I click on Digital Surge on the list of supported exchanges
- When I am taken to signup
- Then that user will have used our full referral link.

## Test steps
-  create an account and take note of affiliate

**Comment by JasminFarhi** [comment]
A referral is done through the API now. 
No way to test this, but make sure that sign up is working.

**Comment by C00p3rb123** [comment]
Blocked by #352 can't test digital surge because I am unable to verify or save mobile number

**Comment by C00p3rb123** [comment]
Unable to sign up to Digital Surge. The issue is in #247

**Comment by TomDodson** [comment]
No longer needed, partner API handles referrals
